hoverctl/cmd/postserveaction.go

```diff
@@ -0,0 +1,75 @@
+package cmd
+
+import (
+	"fmt"
+
+	v2 "github.com/SpectoLabs/hoverfly/core/handlers/v2"
+	"github.com/SpectoLabs/hoverfly/hoverctl/configuration"
+	"github.com/SpectoLabs/hoverfly/hoverctl/wrapper"
+	"github.com/spf13/cobra"
+)
+
+var binary, scriptPath, actionNameToBeSet, actionNameToBeDeleted string
+var delayInMs int
+
+var postServeActionCommand = &cobra.Command{
+	Use:   "post-serve-action",
+	Short: "Get, Set & Delete Hoverfly PostServeAction",
+	Long: `
+Hoverfly PostServeAction can be set using the following flags: 
+	--set --binary --script --delay
+
+Hoverfly PostServeAction can be deleted by passing action name to be deleted:
+	--delete
```

Comment:
it'd be better to be consistent with other hoverctl commands:
```
hoverctl post-serve-action set --name --binary --script --delay
```
and 
```
hoverctl post-serve-action delete --name
```

Reply:
very much required. Thanks for this. incorporated.